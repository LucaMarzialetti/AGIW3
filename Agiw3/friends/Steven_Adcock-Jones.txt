--url_fb https://www.facebook.com/stevejonesllb
--friends
VM45814:3 Ben Hallett
VM45814:3 Steve Swann
VM45814:3 Mark Preston
VM45814:3 Lewis Washington
VM45814:3 Samantha Gould
VM45814:3 Melonie Adcock
VM45814:3 Jasvinder Sahotay
VM45814:3 Louise Rebecca Crosby
VM45814:3 Laura-Pauline Adcock-Jones
VM45814:3 Peter Blackmore
VM45814:3 Amy Catherine
VM45814:3 Elaine Crowshaw
VM45814:3 Kirk Vickers
VM45814:3 Vikkie-lee Girton-Smith
VM45814:3 Christine Smith
VM45814:3 Andy Orpen
VM45814:3 Paul Jones
VM45814:3 Jenna Preston
VM45814:3 Lindsey Washington
VM45814:3 Mat Parry
VM45814:3 Amanda Orpen
VM45814:3 Lisa Porter
VM45814:3 Rachel Ellen
VM45814:3 Darren Malone
VM45814:3 Suze Moussa
VM45814:3 Tim Jones
VM45814:3 Ed Robb
VM45814:3 Kimberly Kemp
VM45814:3 Emily Allen
VM45814:3 Adam Rulewski
VM45814:3 Glenn Orpen
VM45814:3 Shane Vickers
VM45814:3 Ieaun Wales
VM45814:3 Bijoux Chezpeyrat
VM45814:3 Stephanie Ann Metcalfe
VM45814:3 Michael Novak
VM45814:3 Naomi Ryan
VM45814:3 Mike McNally
VM45814:3 Amrik Wahiwala
VM45814:3 Mary Ann Okafor
VM45814:3 Kai von Pahlen
VM45814:3 Sally O'Hare
VM45814:3 Rachael Van Der Toorn
VM45814:3 Richard Morris
VM45814:3 Andrea Thomas
VM45814:3 Mike Evans
VM45814:3 Stacey Bertram
VM45814:3 Melissa Racher
VM45814:3 Mustafa A-s
VM45814:3 Chris Kay
VM45814:3 Christopher Grant Rowe
VM45814:3 Jo Thurlow
VM45814:3 Vicky Reynolds
VM45814:3 Victor O'hara
VM45814:3 Amelia Lees
VM45814:3 Molly Louise
VM45814:3 Leigh Smith
VM45814:3 Patrick Stewart
VM45814:3 Only Fools and Horses Reactions