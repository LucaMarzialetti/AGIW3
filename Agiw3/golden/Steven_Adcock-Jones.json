{
    "PATTERN": {
        "tel": [
            "1897110901",
            "3933333333"
        ],
        "addr": [
            "E708KE",
            "LK8 1HO",
            "SS53SS",
            "F0I7ND",
            "BA88ET"
        ],
        "email": []
    },
    "NER": {
        "LOC": [],
        "ORG": [
            "Kentuckiana Digital Library",
            "Kentuckian Hopkinsville Kentuckian Chas"
        ],
        "PER": ["M. Meacham Hopkinsville"]
    },
    "url": "http://athena.uky.edu/static/news/hopnews/hopnews_KUK-hop1897110901.html"
}